### DISCUSSION TITLE
print dialog box is blank

### DISCUSSION THREAD

**Discussion by kikushyu** [sumatrapdfreader/sumatrapdf#2464] [Q&A]
Hi! I installed sumatra pdf reader 3.3.3.When I want to print a pdf file,I use ctrl+p to open print dialog box
but it's blank.
How to solve?
Thanks!

**Comment by GitHubRulesOK** [comment]
It should not be blank unless there is no printer (this is the result of me using SumatraPDF in a sandbox)
![image](https://user-images.githubusercontent.com/12754913/162230535-c5a3ae6c-368c-4649-97a2-3da9a4b2aa70.png)

Same machine 3.3.3 with just Microsoft printers outside the sandbox
![image](https://user-images.githubusercontent.com/12754913/162231747-3c9127f1-0e91-4b62-8aff-54ad3f08ed75.png)

**Comment by kjk** [comment]
What happens when you try to print from other apps, like notepad?

If the same as sumatra, then it's a windows configuration problem.

If printer dialog from notepad has printers, then it would be something related to Sumatra.

**Reply by kikushyu** [reply]
I found that because I had installed bullzip pdf print(12.2.0.2905)
Uninstall it,sumatar is normal again.

**Comment by GitHubRulesOK** [comment]
It should be good unless you had an installer problem (Not an endorsement as there may be other good alternatives)
 Here I added as only printer and ran a test print no problem
![image](https://user-images.githubusercontent.com/12754913/162346441-0cc09424-bbab-471d-8f6c-9055db24b295.png)
